python/pyspark/shell.py

```diff
@@ -62,6 +62,8 @@
     platform.python_version(),
     platform.python_build()[0],
     platform.python_build()[1]))
+print("Spark context Web UI available at %s" % (sc.uiWebUrl))
+print("Spark context available as 'sc' (master = %s, app id = %s)." % (sc.master, sc.applicationId))
```

Comment:
Can you update shell.R too for SparkR?

Reply:
@HyukjinKwon I have done `shell.R` and manual test result has been update above, please review it when you are free